Lombiq.BaseTheme/Views/Layout.cshtml

```diff
@@ -28,12 +28,12 @@
     <title><shape type="PageTitle"></shape></title>
     <link type="image/x-icon" rel="shortcut icon" href="@Url.Content("~/favicon.ico")" />
 
-    @await RenderSectionAsync("LayoutInjection", required: false)
-
     <script asp-name="bootstrap"></script>
     <script asp-name="@ResourceNames.Helpers"></script>
 
     <resources type="Header"/>
+
+    @await RenderSectionAsync("LayoutInjection", required: false)
```

Comment:
This is a consequential change. Why is it necessary?

Reply:
Without this, the resources are put into the body, so we have 3 link tags in the body, and we can't have link tags in the body since it's an HTML validation error.

Reply:
What causes this change? Or is this like this in `dev` too?

Reply:
Yes, it is also the same in `dev`, the reason why I saw this was the Atata update, where we updated the `HtmlValidatePackageVersion` I think.

Reply:
I see, good then.